### DISCUSSION TITLE
Get Starstruck

### DISCUSSION THREAD

**Discussion by faintout** [Schweinepriester/github-profile-achievements#206] [Q&A] [answered]
Hi all,
for get starstruck need to reach 16 star. If u want u can put star on my readme repository, I will do the same for you.
https://github.com/faintout/ThreeRoomDemo

need only one ,

**Comment by lovecoding-git** [comment, marked as answer]
test

**Reply by sulincix** [reply]
Please leave one here https://github.com/sulincix/debian-subsystem

**Reply by guexs** [reply]
Done https://github.com/wannderlust/wannderlust

**Reply by erikn69** [reply]
@Wannderlust he don't give a star back to anyone

**Reply by WHAHA-HA** [reply]
@sulincix, @guexs, done.
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by rodrigofilhoo** [reply]
if you can help me :3 https://github.com/RodrigoFilho/crudMySql)

**Reply by 1RB** [reply]
https://github.com/1RB/Authworria

**Comment by nel0x** [comment]
Done! :) Please leave one https://github.com/nel0x/arch-installer

**Reply by faintout** [reply]
is ok

**Reply by mezotv** [reply]
Done! here is mine https://github.com/mezotv/TodayIsThatDay

**Reply by sulincix** [reply]
Done here is mine https://github.com/sulincix/debian-subsystem

**Reply by erikn69** [reply]
@sulincix you write done, but you didn't give them any star

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by number0x0001** [comment]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by faintout** [reply]
ok

**Reply by Imperfective** [reply]
I also put star for you!

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by ruymon** [comment]
Hey!

I'll drop you a star!🌟

PS: Here are some tips to make your Repository more attractive in the future :)
- Add good documentation *(here is a website to help you out [README.SO](https://readme.so)*)
- Add a Description to the Project
- Add Tags
- Pin the best projects to your profile.
- Check out other people's profiles and see what they do. (Take @Schweinepriester as an example, with this simple but effective Repo to help out the community).

If followed, the advice above will help you achieve more and more stars!

Good luck!

Regards,

**Reply by ruymon** [reply]
@Schweinepriester If possible, could you set this as the answer for this topic?

**Reply by Schweinepriester** [reply]
@ruymon Nice list!
I could set you as the answer, but I'm not sure if that would even count for Galaxy Brain, I think it counts only if OP does it. But also this discussion is about helping each other, so not sure if there should even be a selected answer.

**Reply by ruymon** [reply]
No worries! Thanks nonetheless. 😉

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by cesarreyes3** [reply]
Please https://github.com/masbug/flysystem-google-drive-ext

**Reply by WHAHA-HA** [reply]
https://github.com/WHAHA-HA/spring-boot-react-ecommerce
thank u

**Comment by Imperfective** [comment]
Done! my repository's https://github.com/Imperfective/Imperfective thx!

**Reply by nel0x** [reply]
Done! Congratulations @Imperfective You made it! :D

**Reply by faintout** [reply]
You made it! :D

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@Imperfective I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by OmarHamdann** [comment]


**Reply by faintout** [reply]
You made it! :D

**Reply by number0x0001** [reply]
Please leave a star here: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by erikn69** [comment]
Done, please help here: https://github.com/erikn69/laravel-google-drive-ext-demo

**Reply by Abhishek10351** [reply]
https://github.com/Abhishek10351/Intensity
A little help 😃

**Reply by erikn69** [reply]
@Abhishek10351 It must be reciprocal, you ask for help but you didn't give it

**Reply by faintout** [reply]
done :D

**Reply by mezotv** [reply]
Done! here is mine https://github.com/mezotv/TodayIsThatDay

**Reply by Abhishek10351** [reply]
Alright @erikn69 done .

**Reply by WHAHA-HA** [reply]
Hello, everyone!
Your repositories was starred. Please leave a star here https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@erikn69 @Abhishek10351 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by AlexRixten** [comment]


**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by Chameleon-Coder** [comment]
https://github.com/Schweinepriester/github-profile-achievements/discussions/152#discussioncomment-3153652

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by guexs** [comment]
I need 1 star! https://github.com/wannderlust/wannderlust

**Reply by faintout** [reply]
done ,please reply

**Reply by guexs** [reply]
Done

**Reply by trentmillar** [reply]
I've been Thunderstruck...nananananananana

**Reply by sulincix** [reply]
done
here is mine https://github.com/sulincix/debian-subsystem

**Reply by csthenry** [reply]
@mezotv Done! here is mine
https://github.com/csthenry/MagicLitePlanet

**Reply by cesarreyes3** [reply]
@Wannderlust done, please https://github.com/masbug/flysystem-google-drive-ext

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@guexs I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@masbug @cesarreyes3 I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by ArielMAJ** [reply]
@mezotv I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by csthenry** [comment]
Done! here is mine
https://github.com/csthenry/MagicLitePlanet

**Reply by guexs** [reply]
Done

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by ArielMAJ** [reply]
@csthenry I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by rawbinary** [comment]


**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by kshitij-halankar** [reply]
@rawbinary Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by ArielMAJ** [reply]
@rawbinary I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by EeeasyCode** [comment]
https://github.com/EeeasyCode/gachon-discord-crawling-bot
come my branch!!

**Reply by WHAHA-HA** [reply]
Done. https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Reply by kshitij-halankar** [reply]
@EeeasyCode Done. Please give me a star for my repo.
https://github.com/kshitij-halankar/TravelR
Much appreciated. Thanks in advance.

**Reply by ArielMAJ** [reply]
@EeeasyCode I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Comment by WHAHA-HA** [comment]
@faintout, already done!
https://github.com/WHAHA-HA/spring-boot-react-ecommerce

**Comment by danielr1996** [comment]
https://github.com/danielr1996/kubectl-action

**Comment by ghost** [comment]


**Reply by kshitij-halankar** [reply]
@AlekseevDanil done

**Comment by number0x0001** [comment]
Please leave a star, thanks: https://github.com/number0x0001/Seanux-Linux-Files-Scripts

**Comment by haithamaouati** [comment]
If my answer helped you, you could mark it as answered. Please.

**Comment by 1RB** [comment]
star this: https://github.com/1RB/Authworria and ill do the same

**Comment by ArielMAJ** [comment]
@faintout I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.
